learn/tanssi/technical.md

```diff
@@ -1,53 +1,95 @@
+![Collators Assignment Algorithm](/images/learn/tanssi/technical/dark-technical-1.png#only-dark)
 
-Polkadot -and the parachains- are built with an open-source blockchain development framework called Substrate, which is bases on Rust programming language.
+### The Role of the Relay Chain {: #relay-chain } 
 
-Substrate makes it easy for teams to develop blockchains, providing pre built core functinalities such as consensus mechanisms, staking & governance, account and assets management, networking and many more ready to use modules and libraries.
+Among many other responsibilities, the relay chain validates and finalizes the blocks produced by any chain participating in the ecosystem (including the ContainerChains and the Tanssi network), storing a small representation of the most recent proof of validity for each block of each chain. This small representation of the proof of validity to be included in the relay chain block is called [candidate receipt](https://polkadot.network/blog/the-path-of-a-parachain-block#candidate-receipts){target=_blank} and is composed of a set of values, including the state root, which can be used to verify state proofs.
 
-The core principles of Substrate are ease of use and performance, allowing teams to create specialized blockchains for any use case, with high level of performance, flexibility, and robustness.
\ No newline at end of file
+As mentioned, the Tanssi network and the ContainerChains are sibling chains with no hierarchical dependency. They are communicated only via the relay chain, and therefore, the relay chain plays a key role in the protocol.
+
+By accessing the data stored in the finalized blocks of the relay chain and cross-referencing headers, the collators can learn their assignation for the session, the ContainerChains can confirm that a certain group of collators from Tanssi has been assigned to them, and Tanssi can verify that the author of a container block was the expected one and reward accordingly.
+
+The Tanssi protocol relays on the relay chain as a means to provide the necessary data to both, Tanssi and its ContainerChains, allowing them to collaborate and validate the correctness of the block production service.
+
+![Relay chain](/images/learn/tanssi/technical/technical-3.png)
+
+### The Role of the Tanssi Network {: #tanssi-newtwork } 
+
+As previously discussed, the Tanssi protocol assigns collators to the Tanssi network itself and the ContainerChains, and the result of this assignment is stored within the chain state.
+
+Another important piece of information that Tanssi stores is the latest header for every ContainerChain. This data is read from the relay chain and, being stored in every Tanssi block, it allows the protocol to keep track of the state in every chain and also to identify and reward accordingly the collator that produced their last block.
+
+![Tanssi Network](/images/learn/tanssi/technical/technical-4.png)
```

Comment:
Better! However, I thought that the point of tanssi reading the container chain state root hash from ther elay chain was to be able to verify that CollatorA (for example) produced the block. So even if you write about this, it would be great to add that in the image also (in the right hand side table)

Reply:
Rearranged the relay chain image, closer to the text that describes it at the beginning, added a line about the collator running also a tanssi node, and the last block producer to the image